coresdk/src/coresdk/text.cpp

```diff
@@ -133,17 +129,27 @@ font load_font(string name, string filename, int size)
     return result;
 }
 
-void _print_strings(void *dest, font fnt, string str, rectangle rc, color fg_clr, color bg_clr, font_alignment flags)
+void font_load_size(string name, int font_size)
```

Comment:
Provide this for `void font_load_size(font fnt, int font_size)` then call that here.

Makes this a one liner.